ProcessSlave.php

```diff
@@ -389,9 +389,9 @@ protected function prepareEnvironment(\React\Http\Request $request)
             $_SERVER['HTTP_' . strtoupper(str_replace('-', '_', $name))] = $value;
         }
 
-        //HTTP_X_REAL_IP is always set, either through ProcessMaster or by a proxy
-        //in front of ProcessMaster.
-        $_SERVER['REMOTE_ADDR'] = $_SERVER['HTTP_X_REAL_IP'];
+        //Using ProcessManager in front of the slaves masks the real/proxy ip
+        $_SERVER['REMOTE_ADDR'] = $_SERVER['HTTP_X_REMOTE_IP'];
```

Comment:
Indeed, that was part of the previous commit, we now don't need to check for a previous header, as we are 'inventing' a new one. BTW I'm open also on a name change of that invented header to 'X_PHPPM_REMOTE_ADDRESS' or something like that